docs/api-and-integration/metadata/jira.md

```diff
@@ -1,55 +1,89 @@
+**Octopus 2019.4** introduced support to integrate Octopus with Jira Cloud and Jira Server. The integration adds links to Jira issues in release details, and release notes (see what's new in a deployment to production for example) to Octopus from Jira. If you're using Jira Cloud, you can also view release and deployment details from Octopus directly in Jira issues, making it possible to see if the issue has been included in any deployments. This builds upon the functionality to [track metadata and work item](/docs/api-and-integration/metadata/index.md) information through your CI/CD pipeline.
 
-This integration uses the [Octopus Deploy plugin for Jira](https://marketplace.atlassian.com/apps/1220376/octopus-deploy-for-jira) and this page describes how to connect and configure the two applications.
+![Octopus release with Jira issues](octopus-release-details.png "width=500")
 
-![Jira Deployments](jira-deployment.png "width=500")
+![Octopus deployment with generated release notes](octopus-release-notes.png "width=500")
 
-:::hint
-The Octopus Deploy plugin for Jira is only compatible with Jira Software Cloud as Jira Server (on-premises) does not support the APIs required to enable this functionality. There is limited support available as noted in our [Jira Server section](#jira-server-on-prem-support).
-:::
+This page described how to configure this functionality for Octopus and Jira.
 
-## Connecting Jira and Octopus Deploy
+* Learn about [Connecting Jira Server and Octopus Deploy](#connecting-jira-server-and-octopus-deploy)
+* Learn about [Connecting Jira Cloud and Octopus Deploy](#connecting-jira-cloud-and-octopus-deploy)
 
-1. Install the Octopus Deploy plugin in Jira.
+---
+
+## Connecting Jira Server and Octopus Deploy
+
+This section describes how to configure Octopus Deploy to connect to Jira Server. In all cases for this integration, connections are always made to Jira (either from Octopus server itself or via links in the browser). There are no cases where Jira connects to Octopus.
+
+This means you can configure any Octopus instance, whether self-hosted or cloud, to use the Jira integration. The only network connectivity requirements are that your Octopus server and your browser can connect to the Jira Base URL discussed below.
+
+If you are using Jira Server, see [Connecting Jira Cloud and Octopus Deploy](#connecting-jira-cloud-and-octopus-deploy).
+
+<!-- I've left this blank so we can review/agree upon the Jira Cloud copy first and then strip it appropriately to suit the Jira Server copy. -->
+
+
+## Connecting Jira Cloud and Octopus Deploy
+
+This section describes how to configure Octopus Deploy to connect to Jira Cloud. In all cases for this integration, connections are always made to Jira (either from Octopus server itself or via links in the browser). There are no cases where Jira connects to Octopus.
```

Comment:
```suggestion
This section describes how to configure Octopus Deploy to connect to Jira Cloud. When configured, this integration adds links in Octopus Deploy (either in the Octopus server itself or via links in the browser) back to your Jira issues. There are no cases where Jira adds links to Octopus.
```
Changed to match the suggestion for the same text in Jira Server (if my interpretation is correct).

Reply:
If the additional features section below moves up above this we can delete: `This section describes how to configure Octopus Deploy to connect to Jira Cloud.` from this paragraph to remove the duplication.